src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserRoleService.cs

```diff
@@ -216,7 +223,7 @@ private async Task<List<string>> GetRequiredGlobalRoleNamesForResourceRoles(Appl
             return requiredGlobalRoleNames;
         }
 
-        private string GetAssociatedGlobalRoleNameForReleaseRole(ReleaseRole role)
+        public string GetAssociatedGlobalRoleNameForReleaseRole(ReleaseRole role)
```

Comment:
We could add some unit tests for this now that it's now public but seems a bit overkill given that it's already tested via other methods already and not doing anything complicated, so happy to leave this.